cmd/bundle/validate.go

```diff
@@ -137,19 +34,35 @@ func newValidateCommand() *cobra.Command {
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
 		ctx := cmd.Context()
 		b, diags := utils.ConfigureBundleWithVariables(cmd)
-		if err := diags.Error(); err != nil {
-			return diags.Error()
+
+		if b == nil {
+			if err := diags.Error(); err != nil {
+				return diags.Error()
+			} else {
+				panic("failed to load bundle")
```

Comment:
Please return `fmt.Errorf` with some description. Always better (and friendlier) than a panic.

Reply:
The reason why I added panic is because it should never happen. If this happens, invariant is violated, and we have a bug in our code, so we should crash immediately. I can change it to `fmt.Errorf` if you think it's more appropriate.